extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -79,13 +79,14 @@ class AmpFitText extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
+  isRelayoutNeeded() {
     return true;
   }
 
   /** @override */
-  isRelayoutNeeded() {
-    return true;
+  prerenderCallback() {
```

Comment:
In other words, you just want to call `this.layoutCallback` from here, right? I think it's better to be explicit here.